src/types/panel-types.ts

```diff
@@ -193,6 +193,7 @@ export class ParsedGenes {
  */
 export class PanelBuilder {
   panelName!: string
+  genes!: Gene[]
```

Comment:
I think we don't use this field anymore. You should be able to delete it. (it's not in the current branch anyway)

Reply:
done